examples/REST/python-flask-starter/app.py

```diff
@@ -0,0 +1,161 @@
+import os, json
+import requests
+from dotenv import load_dotenv
+from flask import Flask, render_template, request, redirect, url_for, session
+
+app = Flask(__name__)
+app.secret_key = "yay v2"
```

Comment:
I would suggest to promote some security hygiene in our examples so secrets should never be stored in files, only consumed from environment